pkg/kubectl/cmd/config/view.go

```diff
@@ -39,20 +37,17 @@ func NewCmdConfigView(out io.Writer, pathOptions *pathOptions) *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:   "view",
-		Short: "displays merged .kubeconfig settings or a specified .kubeconfig file.",
-		Long: `displays merged .kubeconfig settings or a specified .kubeconfig file.
+		Short: "displays .kubeconfig settings or a specified .kubeconfig file.",
+		Long: `displays .kubeconfig settings or a specified .kubeconfig file.
 Examples:
-  // Show merged .kubeconfig settings.
-  $ kubectl config view
-
-  // Show only local ./.kubeconfig settings
-  $ kubectl config view --local`,
+  // Show settings from specified file
+  $ kubectl config view --kubeconfig=path/to/my/.kubeconfig`,
 		Run: func(cmd *cobra.Command, args []string) {
```

Comment:
Should have an example without --kubeconfig specified, since that's the default/most common use case.